webpack.config.extensions.js

```diff
@@ -84,6 +84,10 @@ module.exports = {
 			..._.without( webpackConfig.module.rules, transpileConfig ),
 		],
 	},
+	resolve: {
+		...webpackConfig.resolve,
+		modules: [ ...webpackConfig.resolve.modules, path.resolve( __dirname, '_inc/client' ) ],
```

Comment:
Could also just `[ path.resolve( __dirname, '_inc/client' ), 'node_modules' ]` to keep it simple